src/include/expression/group_marker_expression.h

```diff
@@ -0,0 +1,64 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// group_marker_expression.h
+//
+// Identification: src/include/expression/group_marker_expression.h
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "expression/abstract_expression.h"
+#include "optimizer/group_expression.h"
+#include "util/hash_util.h"
+
+namespace peloton {
+
+namespace executor {
+class ExecutorContext;
+}  // namespace executor
+
+namespace expression {
+
+//===----------------------------------------------------------------------===//
+// GroupMarkerExpression
+//===----------------------------------------------------------------------===//
+
+class GroupMarkerExpression : public AbstractExpression {
```

Comment:
Could you add some brief high-level documentation about GroupMarkerExpression and how it is used?

Reply:
Resolved.